assets/css/style.css

```diff
@@ -0,0 +1,323 @@
+body {
+    margin: 0;
+    font-family: 'Montserrat', sans-serif;
+
+    font-size: 15px;
+    line-height: 1.6;
+    background-color: #576A7D;
+}
+
+*,
+* :after,
+* :before {
+    box-sizing : border-box
+}
+
+h1, h2, h3, h4, h5, h6 {
+    margin: 0;
+}
+
+.container {
+    width: 100%;
+    max-width: 1200px;
+    margin: 0 auto;
+}
+
+.intro {
+    width: 100%;
+    height: 100vh;
+}
+
+
+.header {
+    width: 100%;
+    height: 70px;
+    padding-top: 15px;
+
+    position: absolute;
+    top: 0;
+    left: 0;
+    right: 0;
+    z-index: 1000;
+    background-color: #7F8EA8;
+}
+
+
+.header__inner{
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+}
+
+.nav{
+    font-size: 20px;
+    text-transform: uppercase;
+
+    
+}
+
+.nav__link{
+    display: inline-block;
+    vertical-align: top;
+    margin: 0 20px;
+    position: relative;
+
+    color: #fff;
+    text-decoration: none;
+
+    transition: color 0.1s linear;
+}
+
+
+.nav__link:after{
+    content: "";
+    display: block;
+    width: 100%;
+    height: 3px;
+
+
+    background-color: #fff;
+    opacity: 0;
+
+    position: absolute;
+    top:100%;
+    left: 0;
+    z-index: 1;
+
+    transition: opacity 0.1s linear;
+}
+
+.nav__link:hover{
+    color:#fce38a
+}
+
+
+.nav__link:hover:after{
+    opacity: 1;
+}
+
+
+.footer{

```

Comment:
<img width="353" alt="Screenshot 2022-03-09 at 12 21 35" src="https://user-images.githubusercontent.com/43992068/157431984-8fa75d79-52ca-43c3-a833-18d55f71fcc9.png">

ur footer is not sticked to the bottom.

Reply:
the problem is still here